src/views/TvProgMax/index.jsx

```
@@ -16,7 +16,7 @@ import {
 } from "../../app/features/tvprogmax";
 import { AddTvProgmax, TvProgmaxPlayer } from "../../services/tvprogmax";
 
-const TvProgMax = () => {
+const TvProgMax = ({isDashboard = false}) => {
```

ESLint

[error] 'isDashboard' is missing in props validation

(react/prop-types)